multicluster/config/overlays/leader-ns/coverage/manager_command_patch_coverage.yaml

```diff
@@ -8,6 +8,8 @@ spec:
     spec:
       containers:
         - command: ["/bin/sh"]
-          args: ["-c", "/antrea-mc-controller-coverage -test.run=TestBincoverRunMain -test.coverprofile=antrea-mc-controller.cov.out leader --config=/controller_manager_config.yaml; while true; do sleep 5 & wait $!; done"]
-          name: antrea-mc-controller
+        - args:
+          - antrea-mc-controller-coverage
```

Comment:
Ditto, you need go to antrea/multicluster folder and run `make manifests` to update the MC manifests.

Reply:
Am getting this error:
```
❯ make manifests
antrea/multicluster/bin/controller-gen "crd:allowDangerousTypes=true" rbac:roleName=controller-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xa0117e]

goroutine 103 [running]:
go/types.(*Checker).handleBailout(0xc0008a0400, 0xc003cd3d40)
        /usr/local/go/src/go/types/check.go:367 +0x88
panic({0xbb1600?, 0x1289d00?})
        /usr/local/go/src/runtime/panic.go:770 +0x132
go/types.(*StdSizes).Sizeof(0x0, {0xda4338, 0x12925e0})
        /usr/local/go/src/go/types/sizes.go:228 +0x31e
go/types.(*Config).sizeof(...)
        /usr/local/go/src/go/types/sizes.go:333
go/types.representableConst.func1({0xda4338?, 0x12925e0?})
        /usr/local/go/src/go/types/const.go:76 +0x9e
go/types.representableConst({0xdaa658, 0x125e5e0}, 0xc0008a0400, 0x12925e0, 0xc003cd0ed8)
        /usr/local/go/src/go/types/const.go:106 +0x2c7
go/types.(*Checker).representation(0xc0008a0400, 0xc003cd4b80, 0x12925e0)
        /usr/local/go/src/go/types/const.go:256 +0x65
go/types.(*Checker).representable(0xc0008a0400, 0xc003cd4b80, 0x12925e0)
        /usr/local/go/src/go/types/const.go:239 +0x26
go/types.(*Checker).shift(0xc0008a0400, 0xc003cd4b40, 0xc003cd4b80, {0xda8458, 0xc0012a5d10}, 0x14)
        /usr/local/go/src/go/types/expr.go:650 +0x1eb
go/types.(*Checker).binary(0xc0008a0400, 0xc003cd4b40, {0xda8458, 0xc0012a5d10}, {0xda8968, 0xc0012dfc40}, {0xda8968, 0xc0012dfc60}, 0x14, 0x26aac3)
        /usr/local/go/src/go/types/expr.go:796 +0x150
go/types.(*Checker).exprInternal(0xc0008a0400, 0x0, 0xc003cd4b40, {0xda8458, 0xc0012a5d10}, {0x0, 0x0})
        /usr/local/go/src/go/types/expr.go:1416 +0x206
go/types.(*Checker).rawExpr(0xc0008a0400, 0x0, 0xc003cd4b40, {0xda8458?, 0xc0012a5d10?}, {0x0?, 0x0?}, 0x0)
        /usr/local/go/src/go/types/expr.go:979 +0x19e
go/types.(*Checker).expr(0xc0008a0400, 0x0?, 0xc003cd4b40, {0xda8458?, 0xc0012a5d10?})
        /usr/local/go/src/go/types/expr.go:1513 +0x30
go/types.(*Checker).constDecl(0xc0008a0400, 0xc003c90480, {0x0, 0x0}, {0xda8458, 0xc0012a5d10}, 0x0)
        /usr/local/go/src/go/types/decl.go:488 +0x328
go/types.(*Checker).objDecl(0xc0008a0400, {0xdafc98, 0xc003c90480}, 0x0)
        /usr/local/go/src/go/types/decl.go:191 +0xa49
go/types.(*Checker).ident(0xc0008a0400, 0xc003cd4b00, 0xc0012df6c0, 0x0, 0x0)
        /usr/local/go/src/go/types/typexpr.go:62 +0x250
go/types.(*Checker).exprInternal(0xc0008a0400, 0x0, 0xc003cd4b00, {0xda6f58, 0xc0012df6c0}, {0x0, 0x0})
        /usr/local/go/src/go/types/expr.go:1033 +0x138
go/types.(*Checker).rawExpr(0xc0008a0400, 0x0, 0xc003cd4b00, {0xda6f58?, 0xc0012df6c0?}, {0x0?, 0x0?}, 0x0)
        /usr/local/go/src/go/types/expr.go:979 +0x19e
go/types.(*Checker).expr(0xc0008a0400, 0xc003cd2da0?, 0xc003cd4b00, {0xda6f58?, 0xc0012df6c0?})
        /usr/local/go/src/go/types/expr.go:1513 +0x30
go/types.(*Checker).binary(0xc0008a0400, 0xc003cd4ac0, {0xda8458, 0xc0012a5c50}, {0xda6f58, 0xc0012df6a0}, {0xda6f58, 0xc0012df6c0}, 0xc, 0x26a244)
        /usr/local/go/src/go/types/expr.go:784 +0xcc
go/types.(*Checker).exprInternal(0xc0008a0400, 0x0, 0xc003cd4ac0, {0xda8458, 0xc0012a5c50}, {0x0, 0x0})
        /usr/local/go/src/go/types/expr.go:1416 +0x206
go/types.(*Checker).rawExpr(0xc0008a0400, 0x0, 0xc003cd4ac0, {0xda8458?, 0xc0012a5c50?}, {0x0?, 0x0?}, 0x0)
        /usr/local/go/src/go/types/expr.go:979 +0x19e
go/types.(*Checker).expr(0xc0008a0400, 0xc003c88900?, 0xc003cd4ac0, {0xda8458?, 0xc0012a5c50?})
        /usr/local/go/src/go/types/expr.go:1513 +0x30
go/types.(*Checker).constDecl(0xc0008a0400, 0xc003c889c0, {0x0, 0x0}, {0xda8458, 0xc0012a5c50}, 0x0)
        /usr/local/go/src/go/types/decl.go:488 +0x328
go/types.(*Checker).objDecl(0xc0008a0400, {0xdafc98, 0xc003c889c0}, 0x0)
        /usr/local/go/src/go/types/decl.go:191 +0xa49
go/types.(*Checker).packageObjects(0xc0008a0400)
        /usr/local/go/src/go/types/resolver.go:693 +0x4dd
go/types.(*Checker).checkFiles(0xc0008a0400, {0xc0012b0be0, 0xa, 0xa})
        /usr/local/go/src/go/types/check.go:408 +0x1a5
go/types.(*Checker).Files(...)
        /usr/local/go/src/go/types/check.go:372
sigs.k8s.io/controller-tools/pkg/loader.(*loader).typeCheck(0xc00035cea0, 0xc0026a7840)
        /home/shikhar/go/pkg/mod/sigs.k8s.io/controller-tools@v0.9.0/pkg/loader/loader.go:286 +0x36a
sigs.k8s.io/controller-tools/pkg/loader.(*Package).NeedTypesInfo(0xc0026a7840)
        /home/shikhar/go/pkg/mod/sigs.k8s.io/controller-tools@v0.9.0/pkg/loader/loader.go:99 +0x39
sigs.k8s.io/controller-tools/pkg/loader.(*TypeChecker).check(0xc0012753e0, 0xc0026a7840)
        /home/shikhar/go/pkg/mod/sigs.k8s.io/controller-tools@v0.9.0/pkg/loader/refs.go:268 +0x2b7
sigs.k8s.io/controller-tools/pkg/loader.(*TypeChecker).check.func1(0x5c?)
        /home/shikhar/go/pkg/mod/sigs.k8s.io/controller-tools@v0.9.0/pkg/loader/refs.go:262 +0x53
created by sigs.k8s.io/controller-tools/pkg/loader.(*TypeChecker).check in goroutine 85
        /home/shikhar/go/pkg/mod/sigs.k8s.io/controller-tools@v0.9.0/pkg/loader/refs.go:260 +0x1c5
make: *** [Makefile:50: manifests] Error 2
```